tests/apollo/test_skvbc_reconfiguration.py

```diff
@@ -555,6 +555,228 @@ async def test_addRemove_command(self, bft_network):
             status = cmf_msgs.ReconfigurationResponse.deserialize(r)[0]
             assert status.response.reconfiguration == test_config
 
+    @with_trio
+    @with_bft_network(start_replica_cmd, selected_configs=lambda n, f, c: n == 7)
+    async def test_remove_nodes(self, bft_network):
+        """
+             Sends a addRemove command and checks that new configuration is written to blockchain.
+             Note that in this test we assume no failures and synchronized network.
+             The test does the following:
+             1. A client sends a remove command which will also wedge the system on next next checkpoint
+             2. Validate that all replicas have stopped
+             3. Load  a new configuration to the bft network
+             4. Rerun the cluster with only 4 nodes and make sure they succeed to perform transactions in fast path
+         """
+        bft_network.start_all_replicas()
+        skvbc = kvbc.SimpleKVBCProtocol(bft_network)
+        for i in range(100):
+            await skvbc.write_known_kv()
+        key, val = await skvbc.write_known_kv()
```

Comment:
Why another one?

Reply:
This key val is checked later to validate state consistency after cluster is restarted with new configuration